packages/models-library/src/models_library/resource_tracker.py

```diff
@@ -187,10 +187,33 @@ class SpecificInfo(HardwareInfo):
     to store aws ec2 instance type."""
 
 
+class UnitExtraInfo(BaseModel):
+    """Custom information that is propagated to the frontend. Defined fields are mandatory."""
+
+    CPU: NonNegativeInt
+    RAM: NonNegativeInt
```

Comment:
Please use the ByteSize type here. this is what I already use everywhere. This allows one to define RAM sizes ("10Mib" or "10MB" or 12314321) using units in a string and/or use a a very error prone integer number as now.

Reply:
The user of this is @mguidon and he is expecting Integer type, therefore I would keep it like this in this situation?